programs/create-genesis/genesis_create.cpp

```diff
@@ -1009,6 +1021,104 @@ struct genesis_create::genesis_create_impl final {
+                ("rsharesfn", to_fwide(gp.total_reward_shares2))
+            )
+        );
+
+        fc::flat_map<uint64_t, uint64_t> vote_weights_sum;
+        fc::flat_map<uint64_t, name> authors;
+        db.start_section(gls_post_account_name, N(vote), "voteinfo", _visitor.votes.size());
+        for (const auto& v: _visitor.votes) {
+            std::vector<mvo> delegators;
+            for (const auto& d: v.delegator_vote_interest_rates) {
+                delegators.emplace_back(mvo
+                    ("delegator", get_name(d.account))
+                    ("interest_rate", d.interest_rate)
+                    ("payout_strategy", int(d.payout_strategy))
+                );
+            }
+            auto cid = v.comment;
+            uint64_t w = v.orig_rshares <= 0 ? 0 : v.rshares;
+            vote_weights_sum[cid] += w;
+            if (v.num_changes != 0) {
+                w = 0;
+            } else if (w > 0) {
+                uint64_t auction_window = _visitor.comments[cid].active.auction_window_size;
+                if (v.auction_time != auction_window) {
+                    w = int_arithmetic::safe_prop(w, static_cast<uint64_t>(v.auction_time), auction_window);
+                }
+            }
+            db.insert(pk, authors[cid], mvo
+                ("id", pk)            // uint64_t
+                ("message_id", cid)    // uint64_t
+                ("voter", generate_name(_accs_map[_visitor.acc_id2idx[v.voter]]))         // name
+                ("weight", v.vote_percent)        // int16_t
+                ("time", time_point(v.last_update).time_since_epoch().count())          // uint64_t
+                ("count", v.num_changes)         // uint8_t
+                ("delegators", delegators)    // std::vector<delegate_voter>
+                ("curatorsw", w)     // (fp_cast<fixp_t>(elap_t(sumcuratorsw_delta) * curatorsw_factor)).data(); //base_t
+                ("rshares", to_fbase(v.rshares))       // base_t
+            );
+            pk++;
+        }
+
+        db.start_section(gls_post_account_name, N(message), "message", n);
+        using beneficiary = std::pair<name,uint16_t>;
+        fc::flat_map<std::pair<acc_idx,plk_idx>, uint64_t> post_ids;
+        for (const auto& cp : _visitor.comments) {
+            const auto& c = cp.second;
+            std::vector<beneficiary> beneficiaries;
+            const auto sz = c.active.beneficiaries.size();
+            if (sz) {
+                beneficiaries.reserve(sz);
+                for (const auto& b: c.active.beneficiaries) {
+                    beneficiaries.emplace_back(beneficiary{get_name(b.account), b.weight});
+                }
+            }
+            pk = c.id;
+            db.insert(pk, get_name(c.author), mvo
+                ("id", pk)             // uint64
+                ("permlink", c.permlink.value(_plnk_map))        // string
+                ("ref_block_num", 0)   // uint64
```

Comment:
`ref_block_num` was deleted in https://github.com/GolosChain/golos.contracts/pull/623/files#diff-f003f2f5988fe75d3350500d7f6dfcc6L183

Reply:
fixed in 99f5b4c, also updated to move some fields to `permlink` table from current posting implementation